README.md

```diff
@@ -1,48 +1,105 @@
-# Leukeleu Eslint configuration
+# Leukeleu ESlint configuration
+
+ESlint configuration for Leukeleu projects. To be used in tandem with Prettier.
 
 ## Installing this package
 
-If you already had Eslint installed in your project you can delete this first with `npm uninstall eslint`. There may also be other Eslint packages you want to remove.
+If you already had ESlint installed in your project you can delete this first with `npm uninstall eslint`.
+There may also be other ESlint packages (plugins and configs) you want to remove.
 
 Then install this package as a development dependency `npm i -D @leukeleu/eslint-config`.
 
 ## Adding the config
 
 In the configs folder you can find all the Eslint config files to choose from.
-Select the config you need and add it to your `.eslintrc.js`:
+Select the config you need and add it to your `eslint.config.mjs`.
+
+### Javascript
 
 ```js
-module.exports = {
-  extends: ['@leukeleu/eslint-config/configs/javascript'],
-}
+import eslintConfig from '@leukeleu/eslint-config/configs/javascript'
+
+export default eslintConfig
 ```
 
-## Add script commands to package.json
+### Typescript
+
+```js
+import eslintConfig from '@leukeleu/eslint-config/configs/typescript'
+
+export default eslintConfig
+```
+
+### Vue 2
+
+```js
+import eslintConfig from '@leukeleu/eslint-config/configs/vue2'
 
-To run Eslint on your code, add the following script to your package.json:
+export default eslintConfig
+```
+
+### Vue 3
 
 ```js
-"scripts": {
-  "lint:js": "eslint . --ext .vue,.js,.ts --cache",
-  "lintfix:js": "npm run lint:js -- --fix",
+import eslintConfig from '@leukeleu/eslint-config/configs/vue3'
+
+export default eslintConfig
+```
+
+### Nuxt 3
+
+Nuxt has its own ESLint module that configures ESLint based on the Nuxt project configuration. This generates a config
+file that knows about the Nuxt project structure, layers, auto-imports, etc.
```

Comment:
maybe mention the name / location of that config file here, so you more easily recognize it